helm-charts/daskhub/values.yaml

```diff
@@ -43,14 +43,14 @@ basehub:
         #
         # DASK_GATEWAY__CLUSTER__OPTIONS__IMAGE makes the default worker image
         # match the singleuser image.
-        DASK_GATEWAY__CLUSTER__OPTIONS__IMAGE: "{JUPYTER_IMAGE_SPEC}"
+        DASK_GATEWAY__CLUSTER__OPTIONS__IMAGE: "{{JUPYTER_IMAGE_SPEC}}"
         # DASK_GATEWAY__CLUSTER__OPTIONS__ENVIRONMENT makes some environment
         # variables be copied over to the worker nodes from the user nodes.
-        DASK_GATEWAY__CLUSTER__OPTIONS__ENVIRONMENT: '{"SCRATCH_BUCKET": "$(SCRATCH_BUCKET)", "PANGEO_SCRATCH": "$(PANGEO_SCRATCH)"}'
+        DASK_GATEWAY__CLUSTER__OPTIONS__ENVIRONMENT: '{{"SCRATCH_BUCKET": "$(SCRATCH_BUCKET)", "PANGEO_SCRATCH": "$(PANGEO_SCRATCH)"}}'
         # DASK_DISTRIBUTED__DASHBOARD__LINK makes the suggested link to the
         # dashboard account for the /user/<username>/<server-name> prefix in the path.
         # JUPYTERHUB_SERVICE_PREFIX has leading and trailing slashes as appropriate
-        DASK_DISTRIBUTED__DASHBOARD__LINK: "$(JUPYTERHUB_SERVICE_PREFIX)proxy/{port}/status"
+        DASK_DISTRIBUTED__DASHBOARD__LINK: "{{JUPYTERHUB_SERVICE_PREFIX}}proxy/{{port}}/status"
```

Comment:
Not feeling confident that this change is okay, but could be.

One expansion was a k8s variable expansion, and is now a dask expansion i think.

we got a K8s expansion into kubespawner expansion into dask expansion i think

Reply:
Yeah, am going to poke around and test this out.

Reply:
@consideRatio actually, this is fine, because it was switched to a k8s variable expansion in an intermediate commit in this PR: https://github.com/2i2c-org/infrastructure/pull/2719/commits/d3ae597f9271e4048402fb13ef20c7829d949059. If you look [here](https://github.com/2i2c-org/infrastructure/pull/2719/files#diff-09685dbe199158b7ab0afcf0dd7834d9ce6de3f18045c1e1f318ee03446dd65bR53) you see it's just doing the added escaping.

Reply:
Here's the appropriate output from the JMTE prod hub, running master:

```
(notebook) jovyan@jupyter-yuvipanda:~$ env | grep DASK_GATEWAY__ | sort
DASK_GATEWAY__ADDRESS=http://proxy-public/services/dask-gateway
DASK_GATEWAY__AUTH__TYPE=jupyterhub
DASK_GATEWAY__CLUSTER__OPTIONS__ENVIRONMENT={"SCRATCH_BUCKET": "s3://jmte-scratch/yuvipanda", "PANGEO_SCRATCH": "s3://jmte-scratch/yuvipanda"}
DASK_GATEWAY__CLUSTER__OPTIONS__IMAGE={JUPYTER_IMAGE_SPEC}
DASK_GATEWAY__PROXY_ADDRESS=gateway://traefik-prod-dask-gateway.prod:80
DASK_GATEWAY__PUBLIC_ADDRESS=/services/dask-gateway/
```
And here's the output from staging hub, deployed from this PR:

```
(notebook) jovyan@jupyter-yuvipanda:~$ env | grep DASK_GATEWAY__ | sort
DASK_GATEWAY__ADDRESS=http://proxy-public/services/dask-gateway
DASK_GATEWAY__AUTH__TYPE=jupyterhub
DASK_GATEWAY__CLUSTER__OPTIONS__ENVIRONMENT={"SCRATCH_BUCKET": "s3://jupyter-meets-the-earth-staging-scratch/yuvipanda", "PANGEO_SCRATCH": "s3://jupyter-meets-the-earth-staging-scratch/yuvipanda"}
DASK_GATEWAY__CLUSTER__OPTIONS__IMAGE={JUPYTER_IMAGE_SPEC}
DASK_GATEWAY__PROXY_ADDRESS=gateway://traefik-staging-dask-gateway.staging:80
DASK_GATEWAY__PUBLIC_ADDRESS=/services/dask-gateway/
```

So that looks good to me